src/PhotoLabeler.Components/GridRowTh.razor

```diff
@@ -0,0 +1,10 @@
+﻿<th aria-sort="@(HeaderCell.Order == Entities.Grid.Order.Ascending ? "ascending" : "descending")"
```

Comment:
The "aria-sort" attribute should only be present if the order is defined. If no order is being applied to a certain cell, the attribute must not appear. The idea of this attribute is that the screen reader knows which column we are ordering a table by, and therefore only one of them should have the attribute.

Reply:
Fixed with a c# 8 Switch expression.